fuse/new_orleans_pd.py

```diff
@@ -162,20 +150,26 @@ def fuse_events(pprr_ipm, pprr_csd, cprr, uof, award, lprr, sas):
     officer_number_dict = create_officer_number_dict(pprr_ipm)
     cprr = pd.read_csv(deba.data("clean/cprr_new_orleans_pd_1931_2020.csv"))
     actions = pd.read_csv(deba.data("clean/cprr_actions_new_orleans_pd_1931_2020.csv"))
-    uof = pd.read_csv(deba.data("clean/uof_new_orleans_pd_2012_2019.csv"))
+    uof_officers = pd.read_csv(deba.data("clean/uof_officers_new_orleans_pd_2016_2021.csv"))
+    uof_citizens = pd.read_csv(deba.data("clean/uof_citizens_new_orleans_pd_2016_2021.csv"))
+    uof = pd.read_csv(deba.data("clean/uof_new_orleans_pd_2016_2021.csv"))
+    uof_dfs = [uof_officers, uof_citizens, uof]
+    uof = reduce(
+        lambda left, right: pd.merge(left, right, on="tracking_number"), uof_dfs
+    ).drop_duplicates(subset=["uof_uid"], keep="first")
```

Comment:
Please don't join or merge these tables. Here's what I think should be in the table:
- `uof_officers`: has `tracking_number` and all the officer columns.
- `uof_citizens`: has `tracking_number` and all the citizens columns.
- `uof`: has `tracking_number` of course and all the remaining columns.

No column should be repeated across tables.